README.md

```diff
@@ -134,7 +134,35 @@ You can install Dapr runtime by pulling docker images from a given private regis
 
 ```bash
 # Example of pulling images from a private registry.
-$ dapr init --image-registry example.io/<username>
+dapr init --image-registry example.io/<username>
+```
+
+#### Install in airgap environment
+
+You can install Dapr runtime in airgap (offline) environment using a pre-downloaded [installer bundle](https://github.com/dapr/installer-bundle/releases). You need to download the archived bundle for your OS beforehand (e.g., daprbundle_linux_amd64.tar.gz,) and unpack it. Thereafter use the local Dapr CLI binary in the bundle with `--from-dir` flag in the init command to point to the extracted bundle location to initialize Dapr.
+
+Move to the bundle directory and run the following command:
+
+```bash
+# Initializing dapr in airgap environment
+./dapr init --from-dir .
+```
+
+> For windows, use `.\dapr.exe` to point to the local Dapr CLI binary.
```

Comment:
```bash
# Initializing dapr in airgap Windows environment
.\dapr.exe init --from-dir .
```

Reply:
Suggest making this a cut and paste for Windows, rather than just a comment. Makes the Windows users feel more inclusive!